pr2eus/pr2-interface.l

```diff
@@ -324,6 +325,82 @@ Example: (send self :gripper :rarm :position) => 0.00"
+       (:larm (setq stop-controller "l_arm_controller"))
+       (:larm-loose (setq stop-controller "l_arm_controller_loose"))
+       (:lcam (setq stop-controller "l_forearm_cam_trigger"))
+       (:lgripper (setq stop-controller "l_gripper_controller"))
+       (:lgripper-sensor (setq stop-controller "l_gripper_sensor_controller"))
+       (:tilt (setq stop-controller "laser_tilt_controller"))
+       (:projector (setq stop-controller "projector_controller"))
+       (:projector-trigger (setq stop-controller "projector_trigger"))
+       (:prosilica (setq stop-controller "prosilica_inhibit_projector_controller"))
+       (:rarm (setq stop-controller "r_arm_controller"))
+       (:rarm-loose (setq stop-controller "r_arm_controller_loose"))
+       (:rcam (setq stop-controller "r_forearm_cam_trigger"))
+       (:rgripper (setq stop-controller "r_gripper_controller"))
+       (:rgripper-sensor (setq stop-controller "r_gripper_sensor_controller"))
+       (:torso (setq stop-controller "torso_controller"))
+       )
+     (case start-controller
+       (:base (setq start-controller "base_controller"))
+       (:base-odom (setq start-controller "base_odometry"))
+       (:head-cam-trigger (setq start-controller "head_camera_trigger"))
+       (:head-traj (setq start-controller "head_traj_controller"))
+       (:head-traj-loose (setq start-controller "head_traj_controller_loose"))
+       (:larm (setq start-controller "l_arm_controller"))
+       (:larm-loose (setq start-controller "l_arm_controller_loose"))
+       (:lcam (setq start-controller "l_forearm_cam_trigger"))
+       (:lgripper (setq start-controller "l_gripper_controller"))
+       (:lgripper-sensor (setq start-controller "l_gripper_sensor_controller"))
+       (:tilt (setq start-controller "laser_tilt_controller"))
+       (:projector (setq start-controller "projector_controller"))
+       (:projector-trigger (setq start-controller "projector_trigger"))
+       (:prosilica (setq start-controller "prosilica_inhibit_projector_controller"))
+       (:rarm (setq start-controller "r_arm_controller"))
+       (:rarm-loose (setq start-controller "r_arm_controller_loose"))
+       (:rcam (setq start-controller "r_forearm_cam_trigger"))
+       (:rgripper (setq start-controller "r_gripper_controller"))
+       (:rgripper-sensor (setq start-controller "r_gripper_sensor_controller"))
+       (:torso (setq start-controller "torso_controller"))
+       )
+     (send req :stop_controllers (list stop-controller))
+     (send req :start_controllers (list start-controller))
+     (send req :strictness strictness)
+     (setq res (ros::service-call "/pr2_controller_manager/switch_controller" req))
+     (ros::ros-warn "changing controller from ~A to ~A" stop-controller start-controller)
+     (ros::ros-info "res: ~A" (send res :ok))
+     ))
+  (:start-controller
+   (con)
+   (send self :switch-controller nil con)
+   )
+  (:stop-controller
```

Comment:
What is the practical usage of this method? Please describe as docstring as well as my comment for the previous method.